SofaKernel/modules/SofaCore/src/sofa/core/visual/VisualParams.h

```diff
@@ -140,16 +140,16 @@ class SOFA_CORE_API VisualParams : public ExecParams
     /// Get the projection matrix used to draw the scene. This OpenGL matrix defines the camera coordinate system with respect to the viewport, including perspective if any.
     void getProjectionMatrix( double m[16] ) const { for(unsigned i=0; i<16; i++) m[i] = m_projectionMatrix[i]; }
 
-    SOFA_ATTRIBUTE_DEPRECATED("v21.06 (PR#1649)", "v21.12", "Use with the ModelView and Perspective Matrices instead.")
-    helper::visual::Transformation& sceneTransform() { return m_sceneTransform; }
+    SOFA_ATTRIBUTE_DISABLED("v21.06 (PR#1649)", "v21.12", "Use with the ModelView and Perspective Matrices instead.")
+    helper::visual::Transformation& sceneTransform() = delete;
     
-    SOFA_ATTRIBUTE_DEPRECATED("v21.06 (PR#1649)", "v21.12", "Use with the ModelView and Perspective Matrices instead.")
-    const helper::visual::Transformation& sceneTransform() const { return m_sceneTransform; }
+    //SOFA_ATTRIBUTE_DISABLED("v21.06 (PR#1649)", "v21.12", "Use with the ModelView and Perspective Matrices instead.")
+    //const helper::visual::Transformation& sceneTransform() = delete;
```

Comment:
Why commented?

Reply:
Good question, it was not possible to let the two versions sceneTransform at the same time because the compiler cannot distinguish the two deleted function (and i let it commented to keep a trace)